### DISCUSSION TITLE
Running OpenRadioss with intel MPI on windows on amd cpu or intel cpu

### DISCUSSION THREAD

**Discussion by Calvinsgithub** [OpenRadioss/OpenRadioss#3212] [Q&A]
Hello,
I am a new user of openradioss and I want to use openradioss MPI on windows.I want to confirm whether I can only use intel cpu, but not amd cpu. I would appreciate any guidance.

**Comment by OlivierWienholtz** [comment]
Hello,
It is possible to run OpenRadioss with MPI on AMD machines.
Depending the size of your machine, 
setting KMP_AFFINITY=disabled  environment variable helps.

A part this, no particular issues are known.

Let me know if you have issues.

Best Regards,
Olivier
